dom.go

```diff
@@ -717,21 +720,63 @@ func Rerender(c Component) {
 	if c == nil {
 		panic("vecty: Rerender illegally called with a nil Component argument")
 	}
-	prevRender := c.Context().prevRender
-	if prevRender == nil {
-		panic("vecty: Rerender invoked on Component that has never been rendered")
-	}
 	if c.Context().unmounted {
 		return
 	}
-	nextRender, skip, pendingMounts := renderComponent(c, prevRender)
-	if skip {
-		return
+	batch.add(c)
+}
+
+// batchRenderer handles component re-renders by queueing and deduplicating
+// them, to be rendered on the next animation frame (via requestAnimationFrame).
```

Comment:
Would be good to not mention `deduplicating` here since we don't perform that.

Reply:
Whups

Reply:
This comment is correct now :wink: